plugins/tiddlywiki/filesystem/filesystemadaptor.js

```diff
@@ -43,14 +43,20 @@ Return a fileInfo object for a tiddler, creating it if necessary:
   type: the type of the tiddler file (NOT the type of the tiddler -- see below)
   hasMetaFile: true if the file also has a companion .meta file
 
-The boot process populates $tw.boot.files for each of the tiddler files that it loads. The type is found by looking up the extension in $tw.config.fileExtensionInfo (eg "application/x-tiddler" for ".tid" files).
+The boot process populates $tw.boot.files for each of the tiddler files that it loads.
+
+The type is found by looking up the extension in $tw.config.fileExtensionInfo (eg "application/x-tiddler" for ".tid" files).
 
 It is the responsibility of the filesystem adaptor to update $tw.boot.files for new files that are created.
+
+If `$:/config/FileSystemPaths` exists, we need to test for a new path and delete the old file after saving.
 */
 FileSystemAdaptor.prototype.getTiddlerFileInfo = function(tiddler,callback) {
 	// See if we've already got information about this file
 	var title = tiddler.fields.title,
-		fileInfo = $tw.boot.files[title];
+		fileInfo = $tw.boot.files[title],
+		fileSystemPaths = this.wiki.tiddlerExists("$:/config/FileSystemPaths"),
+		options = {};
```

Comment:
options is initialized as an empty Object.  In line 79 it is set to null. ... hmmm